platform/representation/src/gestures/DropGesture.js

```diff
@@ -165,16 +165,18 @@ define(
                     if (shouldCreateVirtualPanel(domainObject, selectedObject)){
```

Comment:
Yeah I totally agree. Actually the intention here was not to do the dropGesture refactoring just yet. I intend to defer it for #636. I did some limited refactoring that was necessitated by the changes to navigation. I will remove the commended code though, that was just an oversight.